bundles/org.openhab.binding.miele/src/main/resources/OH-INF/thing/coffeemachine.xml

```diff
@@ -12,6 +12,7 @@
 
 		<label>Coffee Machine</label>
 		<description>This is a Miele@home compatible coffee machine</description>
+		<semantic-equipment-tag>WhiteGood</semantic-equipment-tag>
```

Comment:
Out of context for this PR, but mentioning here anyway: I guess that even if a bulky and heavy coffee machine can be considered as white good, a coffee machine might qualify to have its own equipment tag as something unique, whereas `WhiteGood` is quite generic?

Reply:
Indeed. There are four or five bindings that expose coffee machines..

Reply:
But it is probably worth keeping in mind that normal homes will only have one single instance of cook top, washing machine, dryer, refrigerator, kitchen hood, etc. and indeed coffee machine. This means the child levels of the 'browsing tree' (e.g. washing machine) will always only contain one entry; which makes little sense.

By contrast the parent level of the browsing tree 'White goods' (a.k.a. kitchen appliances) would contain a meaningful group of Things.

Reply:
I need to catch up on the recent improvements of the documentation. I guess there is also a description of what semantic tags can be used for. You mention grouping. I guess they could also be used e.g. by a coffee machine (or washing machine, etc.) widget to automatically detect relevant items? And rules can use them to find specific stuff. Anything else?

I guess by the definition of "more than one is unlikely to exist in a home", we should remove:
- Projector
- GarageDoor (and other specialized doors).
- most WhiteGood childs as you already mentioned

Or?

Reply:
That is my opinion indeed. But probably others would have other opinions.

Reply:
@jlaur shall we mark this as resolved? If you want to open a PR to create `Equipment.CoffeeMaker` (as a child of `WhiteGood`) you are welcome so to do. And we can adapt this binding when that other PR is merged.

Reply:
I have created openhab/openhab-core#4726.